tests/integration/conftest.py

```diff
@@ -17,6 +17,11 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
+# Test tiers:
+#   * Tier 1: Normal test case and should never fail.
+#   * Tier 2: Decorative test case and should never fail.
+#   * Tier 3: Test case might randomly fails.
```

Comment:
A test should only be in tier 3 temporarily, shouldn't it? If it is the case, how about calling this "Temporarily unstable test case that somtimes fails".